src/p2p/net_node.inl

```diff
@@ -1671,6 +1673,27 @@ namespace nodetool
     });
     return ret;
   }
+
+  //-----------------------------------------------------------------------------------
+  template<class t_payload_net_handler>
+  bool node_server<t_payload_net_handler>::find_connection_id_by_peer(const peerlist_entry &pe, boost::uuids::uuid& conn_id)
```

Comment:
1. Any chance we're duplicating code here? I have feeling I saw this before
2. explicit capturing please

Reply:
1 - deleted old function;
2 - done;

Reply:
squashed into one commit